plugins/dfu/dfu-common.h

```diff
@@ -171,6 +172,7 @@ typedef enum {
 	DFU_VERSION_DFU_1_0			= 0x0100,
 	DFU_VERSION_DFU_1_1			= 0x0110,
 	DFU_VERSION_DFUSE			= 0x011a,
```

Comment:
So shouldn't this be switched to `DFU_VERSION_STM`?

Reply:
I flip-flopped on this. STM do actual call the version number DfuSe, and it's a defined value. I think adding a comment will make this clearer.